build.gradle

```diff
@@ -0,0 +1,28 @@
+plugins {
+	id 'java'
+	id 'org.springframework.boot' version '3.4.1'
+	id 'io.spring.dependency-management' version '1.1.7'
+}
+
+group = 'com.delo'
+version = '0.0.1-SNAPSHOT'
```

Comment:
What is meaning of snapshot in the version here ?

Reply:
It refers to a development or pre-release version that is being actively worked on. It represents the current state of the software at a particular point in time.